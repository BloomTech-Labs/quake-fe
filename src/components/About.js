import React from "react";
import aboutMap from "../images/about-us-map.png";
import aboutVerticalImg from "../images/about-us-vertical.png"
const About = () => {
  return (
    <>
      <div className="spacer" />
      <main className="about-main">
        <section className="about-links-container">
          <h1>We are FaultLine</h1>
          <aside className="about-team-img" />
          <section className="about-team-links">
            <a href={"/"} className="ej">Eddie</a>
            <a href={"/"} className="ls">Laura</a>
            <a href={"/"} className="kd">Ken</a>
            <a href={"/"} className="jb">Jeff</a>
            <a href={"/"} className="pj">Pete</a>
            <a href={"/"} className="ch">Cesar</a>
            <a href={"/"} className="jg">Jeremy</a>
            <a href={"/"} className="nw">Nicole</a>
          </section>
        </section>

        <section className="about-map-container">
          <aside className="about-horizontalbg">
            <h1>We live in 8 different states</h1>
          </aside>
          <aside className="about-map-verticalbg">
            <img className="imgMap" src={aboutMap} alt="map of United States"/>
            <img className="vertical-accent" src={aboutVerticalImg} />
          </aside>
        </section>
      </main>
    </>
  );
};

export default About;
