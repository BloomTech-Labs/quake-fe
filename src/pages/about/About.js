import React from "react";
import colorfulMap from "../../images/graphics/map-color.svg";
import mapAccent from "../../images/graphics/map-accent.png";
import pieChart from "../../images/graphics/pie-chart-about.svg";
import girlDog from "../../images/graphics/girl-dog-about.svg";
import girlCat from "../../images/graphics/gitl-cat-about.svg";
import fatherKid from "../../images/graphics/father-daughter-about.svg";
const About = () => {
  return (
    <div className="main-container scroll">
      <main className="about-main">
        <section className="about-links-info-container">
          <h1 className="header-about">We are FaultLine</h1>
          <aside className="about-team-img" />
          <section className="about-team-links">
            <aside className="links-row-one">
              <a href={"https://github.com/petedram"}>Pete</a>
              <a href={"https://github.com/jjbreig5909"}>Jeff</a>
              <a href={"https://github.com/cesarhj19"}>Cesar</a>
            </aside>
            <aside className="links-row-two">
              <a href={"https://github.com/Nicci498"}>Nicole</a>
              <a href={"https://github.com/l8nightswithJS"}>Eddie</a>
              <a href={"https://www.linkedin.com/"}>Jeremy</a>
              <a href={"https://www.laurasuyasa.com/"}>Laura</a>
              <a href={"https://github.com/diehlkj"}>Ken</a>
            </aside>
          </section>
        </section>

        <section className="map-states-container">
          <h1 className="header-about">We live in 4 different states</h1>
          <img
            className="colorful-map"
            src={colorfulMap}
            alt="colorful map with teams location marked"
          />
          <img
            className="map-accent"
            src={mapAccent}
            alt="map accent background"
          />
        </section>

        <section className="pie-chart-section-container">
          <h1 className="header-about">Our Jobs Are:</h1>
          <aside className="bullets-pie-chart-container">
            <img
              className="pie-chart"
              src={pieChart}
              alt="colorful map with teams location marked"
            />
            <aside className="list-container">
              <ul>
                <li className="orange-bullet">5 Developers</li>
                <li className="pink-bullet">2 Designers</li>
                <li className="turquoise-bullet">1 Team Leader</li>
              </ul>
            </aside>
          </aside>
        </section>
        <section className="between-us-container">
          <h1 className="header-about">Between Us We Have:</h1>
          <aside className="dogs-cats-container">
            <img
              className="imgDog"
              src={girlDog}
              alt="girl standing next to small dog"
            />
            <h1 className="header-about">3 Dogs</h1>
          </aside>
          <aside className="kids-container">
            <h1 className="header-about">6 Kids</h1>
            <img src={fatherKid} alt="girl standing next to small dog" />
          </aside>
          <aside className="dogs-cats-container">
            <img
              className="imgCat"
              src={girlCat}
              alt="girl sitting cross legged next to pink cat"
            />
            <h1 className="header-about">8 Cats</h1>
          </aside>
          <p>
            We are passionate about our product and we hope that you love it
            too!
          </p>
        </section>
      </main>
    </div>
  );
};

export default About;
