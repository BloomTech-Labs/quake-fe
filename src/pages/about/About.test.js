import React from 'react'
import About from './About'
import renderer from 'react-test-renderer'

describe('About Snapshot', () => {
let aboutComponent = renderer.create(<About/>)

  it('renders matching snapshot', () => {
    expect(aboutComponent.toJSON()).toMatchSnapshot();
  })
})