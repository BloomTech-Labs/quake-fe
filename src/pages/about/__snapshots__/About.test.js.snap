// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About Snapshot renders matching snapshot 1`] = `
<div
  className="main-container scroll"
>
  <main
    className="about-main"
  >
    <section
      className="about-links-info-container"
    >
      <h1
        className="header-about"
      >
        We are FaultLine
      </h1>
      <aside
        className="about-team-img"
      />
      <section
        className="about-team-links"
      >
        <aside
          className="links-row-one"
        >
          <a
            href="https://github.com/petedram"
          >
            Pete
          </a>
          <a
            href="https://github.com/jjbreig5909"
          >
            Jeff
          </a>
          <a
            href="https://github.com/cesarhj19"
          >
            Cesar
          </a>
        </aside>
        <aside
          className="links-row-two"
        >
          <a
            href="https://github.com/Nicci498"
          >
            Nicole
          </a>
          <a
            href="https://github.com/l8nightswithJS"
          >
            Eddie
          </a>
          <a
            href="https://www.linkedin.com/"
          >
            Jeremy
          </a>
          <a
            href="https://www.laurasuyasa.com/"
          >
            Laura
          </a>
          <a
            href="https://github.com/diehlkj"
          >
            Ken
          </a>
        </aside>
      </section>
    </section>
    <section
      className="map-states-container"
    >
      <h1
        className="header-about"
      >
        We live in 4 different states
      </h1>
      <img
        alt="colorful map with teams location marked"
        className="colorful-map"
        src="map-color.svg"
      />
      <img
        alt="map accent background"
        className="map-accent"
        src="map-accent.png"
      />
    </section>
    <section
      className="pie-chart-section-container"
    >
      <h1
        className="header-about"
      >
        Our Jobs Are:
      </h1>
      <aside
        className="bullets-pie-chart-container"
      >
        <img
          alt="colorful map with teams location marked"
          className="pie-chart"
          src="pie-chart-about.svg"
        />
        <aside
          className="list-container"
        >
          <ul>
            <li
              className="orange-bullet"
            >
              5 Developers
            </li>
            <li
              className="pink-bullet"
            >
              2 Designers
            </li>
            <li
              className="turquoise-bullet"
            >
              1 Team Leader
            </li>
          </ul>
        </aside>
      </aside>
    </section>
    <section
      className="between-us-container"
    >
      <h1
        className="header-about"
      >
        Between Us We Have:
      </h1>
      <aside
        className="dogs-cats-container"
      >
        <img
          alt="girl standing next to small dog"
          className="imgDog"
          src="girl-dog-about.svg"
        />
        <h1
          className="header-about"
        >
          3 Dogs
        </h1>
      </aside>
      <aside
        className="kids-container"
      >
        <h1
          className="header-about"
        >
          6 Kids
        </h1>
        <img
          alt="girl standing next to small dog"
          src="father-daughter-about.svg"
        />
      </aside>
      <aside
        className="dogs-cats-container"
      >
        <img
          alt="girl sitting cross legged next to pink cat"
          className="imgCat"
          src="gitl-cat-about.svg"
        />
        <h1
          className="header-about"
        >
          8 Cats
        </h1>
      </aside>
      <p>
        We are passionate about our product and we hope that you love it too!
      </p>
    </section>
  </main>
</div>
`;
